src/wordpress.js

```diff
@@ -23,7 +23,10 @@ const fixInternalURLs = (content, { baseURL = WORDPRESS_BASE_URL } = {}) => {
 
   return content.replace(
     // Make sure the URL pattern accepts both http:// and https://.
-    new RegExp(baseURL.replace(/^https?:/, 'https?:'), 'g'),
+    new RegExp(
+      baseURL.replace(/^https?:/, 'https?:') + '(?!/wp-content/)',
```

Comment:
Could this be a template string?

Reply:
Yes, that's what eslint said too :p I rewrote it with `[].join('')` to add comments above each part.